gradle.properties

```diff
@@ -1,9 +1,9 @@
-VERSION_NAME=5.2.0
+VERSION_NAME=6.0.0
```

Comment:
Concern: Changing the major version of an SDK is kind of a big deal and not something to do lightly. This issue is quite edge case (we've had one customer hit it in the past 4 years) and there is an easy workaround, so it doesn't feel worth it to me.